Server/rainbow-server/Sources/Application/Constant.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Constant.swift
+//  Application
+//
+//  Created by Sanjeev Ghimire on 5/9/18.
+//
+
+import Foundation
+
+public struct Constant {
+    
+    public static let NOTIFICATION_MESSAGE_KNOCKED_FROM_TOP: String = "You have been knocked from the top"
```

Comment:
how about:

"You have been knocked from the top spot on the leaderboard. Play again to reclaim your spot!"

Reply:
We should get @anamita to confirm these.